### PR TITLE
Group and Prioritize Add-ons list.

### PR DESCRIPTION
**Pull request by kjohnson**
- Adds category grouping to the add-ons list (feed).
- De-prioritizes the order of already installed add-ons.
- De-prioritizes the CRM or Newsletter groups if a grouped add-on is already installed.

### CODE REVIEW

**Comment by kjohnson** [comment]
| Core (Only) | File Uploads (Builder) | MailChimp (Newsletter) |
| --- | --- | --- |
| image | image | image |

File: includes/addons.json [hunk 1]
```diff
@@ -0,0 +1,373 @@
+{
+  "addons": [
+    {
+      "title": "Layout and Styles",
+      "image": "assets/img/add-ons/layout-styles.png",
+      "content": "Easily create multi-column form layouts and beautifully styled form elements without mastering CSS.",
+      "link": "https://ninjaforms.com/extensions/layout-styles/?utm_medium=plugin&utm_source=plugin-addons-page&utm_campaign=Ninja+Forms+Addons+Page&utm_content=Layout+and+Styles",
+      "plugin": "ninja-forms-layout-styles/ninja-forms-style.php",
+      "docs": "https://ninjaforms.com/docs/layouts-and-styles/?utm_medium=plugin&utm_source=plugin-addons-page&utm_campaign=Ninja+Forms+Addons+Page&utm_content=Layout+and+Styles+Docs",
+      "version": "3.0.18",
```
Comment (Much2tall) [inline includes/addons.json]: Who do these include version numbers?
Will need to be maintained in future updates if we leave them. (Duplicated work.)

Comment (kjohnson) [reply includes/addons.json]: The feed was originally going to be sourced from the NinjaForms server, so the idea was that we could ID updates - even without license keys. However, the feed was added to the plugin, so this needs to be revisited.

**Comment by Much2tall** [comment]
Closing in favor of additional changes before resubmission of PR.

**Pull request closed without merging**